docs/sample-out/section=123456789/assignments/2020-09-17-15-04-23.csv

```diff
@@ -1,2 +1,2 @@
 ﻿SourceSystemIdentifier,SourceSystem,Title,AssignmentCategory,AssignmentDescription,StartDateTime,EndDateTime,DueDateTime,SubmissionType,MaxPoints,LMSSectionSourceSystemIdentifier,CreateDate,LastModifiedDate,SourceCreateDate,SourceLastModifiedDate
-2942251012,BestLMS, Redundancy in War and Peace,Assignment, Describe with statistical examples how Tolstoy could have written a shorter novel by not repeating his theme so frequently., 2021-04-01 09:00:00, 2021-05-01 00:00:00, 2021-05-01 00:00:00,online_upload,100,123456789,2020-08-20 12:34:50,2020-08-20 12:34:50,,
+2942251012,BestLMS," Redundancy in War and Peace",Assignment," Describe with statistical examples how Tolstoy could have written a shorter novel by not repeating his theme so frequently."," 2021-04-01 09:00:00"," 2021-05-01 00:00:00"," 2021-05-01 00:00:00",online_upload,100,123456789,2020-08-20 12:34:50,2020-08-20 12:34:50,,
```

Comment:
Quotation marks don't hurt here, but they also weren't necessary. Only necessary if there is a comma in the string. It is a little weird to have the space at the beginning, so I'm going to remove it:

```suggestion
2942251012,BestLMS,"Redundancy in War and Peace",Assignment,"Describe with statistical examples how Tolstoy could have written a shorter novel by not repeating his theme so frequently.","2021-04-01 09:00:00"," 2021-05-01 00:00:00"," 2021-05-01 00:00:00",online_upload,100,123456789,2020-08-20 12:34:50,2020-08-20 12:34:50,,
```